src/caretogether-pwa/src/App.tsx

```diff
@@ -130,19 +130,21 @@ function App() {
       <CssBaseline />
       <Router>
         <Header open={open} handleDrawerOpen={handleDrawerOpen} />
-        {isMobile ? null :
+        {isMobile ? null : (
```

Comment:
Looks like we might want to get some file format convention going with prettier, a VS Code extension, or perhaps use a githook oncommit

Reply:
I don't understand what you are proposing. I use prettier, but at the time working in this app, I had disabled it.
I don't see a large change in the code you mentioned. I think the parentheses were recommended.

Do you want me to implement `prettier` that formats code on save, commit, push, or anything like that?

Reply:
You're right, it's not a large change. File formatting differences can make it difficult to find the intentional code changes. I wanted to bring up the idea to @LarsKemmann. I wouldn't recommend any changes until Lars jumps in.

Reply:
I agree, it would be nice to normally have formatting differences in dedicated PRs for that. Not a problem here and I agree with the changes that were made being recommended. I've created #207 to track implementing Prettier to format code on save.